modules/ROOT/pages/runtime-fabric-setup.adoc

```diff
@@ -0,0 +1,46 @@
+= Runtime Fabric Setup
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Verify Supported Mule Versions for Runtime Fabric
+
+Anypoint Visualizer supports the following Mule versions for Runtime Fabric deployments:
+
+* Mule 3.8.7
+* Mule 3.9.1+
+* Mule 4.1.2+
+* Mule 4.2.0
+
+Follow the instructions for xref:runtime-fabric::install-create-rtf-arm.adoc[setting up Runtime Fabric].
+After you install Runtime Fabric, Anypoint Visualizer can display applications that are running and deployed to Runtime Fabric.
+
+Anypoint Visualizer relies on a header injection to display connections between services correctly.
+
+== Enable Header Injection
+
+Anypoint Visualizer requires an additional header injection for Runtime Fabric deployments. However, due to a potential performance impact for Mule 4, header injection is disabled by default.
+For Runtime Fabric, you must enable the header injection at the app level. You can enable the header injection in the *Properties* tab for an app.
+
+. If you have already deployed apps to Runtime Fabric:
+.. Click *Applications* in the menu on the left of Runtime Manager.
+.. Select the app for which to enable the header injection, and click *Manage Application*.
+. If you are deploying an app for the first time:
+.. On the deployment page, click *Deploy application*.
+. Click the *Properties* tab, and set the following property in the app: `anypoint.platform.config.analytics.agent.header_injection.disabled=false`.
+
+You must set the property to `false` as shown in the example to enable header injection, since the property is disabled by default.
```

Comment:
```suggestion
You must set the property to `false`, as shown in the example to enable header injection, because the property is disabled by default.
```